src/Sidekick.Business/Apis/Poe/Parser/Patterns/ParserPatterns.cs

```diff
@@ -108,12 +108,12 @@ private void InitSockets()
 
         private void InitInfluences()
         {
-            Crusader = new Regex($"[\\r\\n]+{Regex.Escape(languageProvider.Language.InfluenceCrusader)}");
-            Elder = new Regex($"[\\r\\n]+{Regex.Escape(languageProvider.Language.InfluenceElder)}");
-            Hunter = new Regex($"[\\r\\n]+{Regex.Escape(languageProvider.Language.InfluenceHunter)}");
-            Redeemer = new Regex($"[\\r\\n]+{Regex.Escape(languageProvider.Language.InfluenceRedeemer)}");
-            Shaper = new Regex($"[\\r\\n]+{Regex.Escape(languageProvider.Language.InfluenceShaper)}");
-            Warlord = new Regex($"[\\r\\n]+{Regex.Escape(languageProvider.Language.InfluenceWarlord)}");
```

Comment:
This change breaks on this item. I was making sure this was the first text on the line. Maybe putting a ^ at the beginning of the regex would be sufficient.
```
Rarity: Rare
Entropy Heart
Small Cluster Jewel
--------
Requirements:
Level: 54
--------
Item Level: 72
--------
Adds 3 Passive Skills (enchant)
Added Small Passive Skills grant: 15% increased Armour (enchant)
--------
Added Small Passive Skills also grant: +3 to Intelligence
Added Small Passive Skills also grant: +4% to Lightning Resistance
Added Small Passive Skills also grant: +7 to Maximum Life
1 Added Passive Skill is Dragon Hunter
--------
Place into an allocated Small, Medium or Large Jewel Socket on the Passive Skill Tree. Added passives do not interact with jewel radiuses. Right click to remove from the Socket.
```

Reply:
Nice catch, changed to match the start of line first.